### ISSUE TITLE
Error During Startup - "can not import leo.plugins.writers.plugins"

### ISSUE THREAD

**Issue by tbpassin** [leo-editor/leo-editor#3045]
I have started getting the following error message during startup:
```
Traceback (most recent call last):

  File "c:\Tom\git\leo-editor\leo\core\leoApp.py", line 2480, in createWritersData
    m = importlib.import_module(f"leo.plugins.writers.{sfn[:-3]}")

  File "C:\Users\tom\AppData\Local\Programs\Python\Python310\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)

  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import

  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load

  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked

ModuleNotFoundError: No module named 'leo.plugins.writers.plug'

can not import leo.plugins.writers.plugins
can not import leo.plugins.importers.plug
[
         'C:/Users/tom/.leo/plugins'
]
```

I am able to successfully import these writers plugins in a Python REPL session outside of Leo.  I added a print statement to look at the import paths, and they look normal to me.

I have not noticed any operational faults with saving outlines with external files, but I don't have any `@auto` files.  The readme file in the plugins/writers directory says they apply to `@auto` files.

I'm not sure when this started.  I just noticed it this morning, but it seems to have been in devel before now.

**Comment by edreamleo** [comment]
@tbpassin Thanks for this report. The first and last entries in the traceback shows the probable cause:
```console
File "c:\Tom\git\leo-editor\leo\core\leoApp.py", line 2480, in createWritersData
    m = importlib.import_module(f"leo.plugins.writers.{sfn[:-3]}")
...
ModuleNotFoundError: No module named 'leo.plugins.writers.plug'

can not import leo.plugins.writers.plugins
can not import leo.plugins.importers.plug
[
         'C:/Users/tom/.leo/plugins'
]
```
Presumably the last three characters of the *module* name `leo.plugins.writers.plugins` are being truncated.

This is surprisingly difficult code, but ensuring that sfn endswith ".py" would probably be a good enough guard.

**Comment by edreamleo** [comment]
@tbpassin This issue is my responsibility. I'll submit a PR asap.

**Comment by tbpassin** [comment]
"Presumably the last three characters of the module name leo.plugins.writers.plugins are being truncated."

Looking at my printed diagnostics, the failure may be only in this path:

`leo.plugins.writers.plug`

Here is the complete listing from my printout inside the import loop (the path printouts are intermixed with the normal exception reports to that log tab.  My print outs of the import paths are prefixed by "===="):

```
==== leo.plugins.writers.plug
Traceback (most recent call last):
  File "c:\Tom\git\leo-editor\leo\core\leoApp.py", line 2480, in createWritersData
    m = importlib.import_module(f"leo.plugins.writers.{sfn[:-3]}")
  File "C:\Users\tom\AppData\Local\Programs\Python\Python310\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'leo.plugins.writers.plug'
can not import leo.plugins.writers.plugins
==== leo.plugins.writers.basewriter
==== leo.plugins.writers.ctext
==== leo.plugins.writers.dart
==== leo.plugins.writers.leo_rst
==== leo.plugins.writers.markdown
==== leo.plugins.writers.org
==== leo.plugins.writers.otl
==== leo.plugins.writers.treepad
can not import leo.plugins.importers.plug
```

It seems that there is only one spurous failed import, and that is intended to be for an entire package that doesn't exist: `leo.plugins.writers.plug`

**Comment by edreamleo** [comment]
Fixed via PR #3046.
